WebRoot/js/zimbraMail/abook/view/ZmContactSearch.js

```diff
@@ -442,9 +439,11 @@ function(htmlArr, idx, contact, field, colIdx, params) {
 	if (field == ZmItem.F_TYPE) {
 		htmlArr[idx++] = AjxImg.getImageHtml(contact.getIcon());
```

Comment:
Can you check if contact's image shows up when you upload an image for a contact instead of just default icon?